evm/build.gradle

```diff
@@ -43,9 +46,11 @@ dependencies {
   implementation 'org.apache.tuweni:tuweni-bytes'
   implementation 'org.hyperledger.besu:arithmetic'
   implementation 'org.hyperledger.besu:bls12-381'
+  implementation 'tech.pegasys:jc-kzg-4844:0.1.0'
```

Comment:
versions should not be in module build files.  Enumerate them in `versions.gradle`